app/Controllers/statsController.php

```diff
@@ -49,22 +49,43 @@ private function convertToPieSerie($data) {
 	 *
 	 * It displays the statistic main page.
 	 * The values computed to display the page are:
-	 *   - repartition of read/unread/favorite/not favorite
-	 *   - number of article per day
-	 *   - number of feed by category
-	 *   - number of article by category
-	 *   - list of most prolific feed
+	 *   - repartition of read/unread/favorite/not favorite (repartition)
```

Comment:
Btw, if you're touching all these lines anyway, you might as well replace it all with a correct word like `distribution` instead. Répartition is French.

Reply:
Repartition is also English ;)
https://www.wordreference.com/definition/repartition

Reply:
I'm well aware that it is *a* word. It's a false friend. La répartition des articles = distribution of articles (or: article distribution).

Other words that can be used depending a bit on context are breakdown and division. E.g., breakdown by hour instead of distribution by hour.

https://fr.wikipedia.org/wiki/Pyramide_des_%C3%A2ges
> La pyramide des âges représente la répartition par sexe et âge de la population à un instant donné.

https://en.wikipedia.org/wiki/Population_pyramid
> Population distribution by age and sex for Angola in 2005

Or more directly in a translation dictionary:
https://en.pons.com/translate/french-english/r%C3%A9partition#result-sig-61618324-0

> la répartition des revenus en France
    income distribution in France

Whether the word exists is not in question. You won't find this sense in COCA (and yes, I checked to be sure; mainly computers, a bit repartition of Jerusalem/Ireland).

Reply:
let's do this i18n thing in another PR. I have also other things in mind to correct

Reply:
No problem.